homeassistant/components/aurora_abb_powerone/sensor.py

```diff
@@ -45,6 +46,7 @@
     SensorEntityDescription(
         key="temp",
         device_class=DEVICE_CLASS_TEMPERATURE,
+        entity_category=ENTITY_CATEGORY_DIAGNOSTIC,
```

Comment:
Could you use the new `EntityCategory` enum?
```suggestion
        entity_category=EntityCategory.DIAGNOSTIC,
```

Reply:
Agreed. Done. New PR for the additional enums is #61245